pkg/okteto/auth.go

```diff
@@ -66,34 +112,19 @@ func Auth(ctx context.Context, code, url string) (*types.User, error) {
 		if err != nil && oktetoErrors.IsX509(err) {
 			return nil, err
 		}
-
-		return nil, fmt.Errorf("authentication error, please try again")
+		err = newAuthenticationErr(err)
+		return nil, err
 	}
 
 	return user, nil
 }
 
 func (c *OktetoClient) authUser(ctx context.Context, code string) (*types.User, error) {
-	var mutation struct {
-		User struct {
-			Id              graphql.String
-			Name            graphql.String
-			Namespace       graphql.String
-			Email           graphql.String
-			ExternalID      graphql.String `graphql:"externalID"`
-			Token           graphql.String
-			New             graphql.Boolean
-			Registry        graphql.String
-			Buildkit        graphql.String
-			Certificate     graphql.String
-			GlobalNamespace graphql.String  `graphql:"globalNamespace"`
-			Analytics       graphql.Boolean `graphql:"telemetryEnabled"`
-		} `graphql:"auth(code: $code, source: $source)"`
-	}
+	var mutation authMutationStruct
```

Comment:
```suggestion
	var authMutation authMutationStruct
```
I think thgis improves reading it adding more context